### PR TITLE
Fix feedback pdf table column widths

### PR DESCRIPTION
**Pull request by hammerfunk**

### CODE REVIEW

File: app/pdf_generators/feedback_pdfs/general/data_pointer.rb [hunk 1]
```diff
@@ -25,11 +25,15 @@ def feedback_entries
 
   def render_data!
     table_items = feedback_entries
-    render_headers(feedback_table_headers)
+    render_headers(feedback_table_headers, {
```
Comment (matthewford) [inline app/pdf_generators/feedback_pdfs/general/data_pointer.rb]: Redundant curly braces around a hash parameter.

File: app/pdf_generators/feedback_pdfs/general/data_pointer.rb [hunk 2]
```diff
@@ -25,11 +25,15 @@ def feedback_entries
...
+      0 => 130,
```
Comment (matthewford) [inline app/pdf_generators/feedback_pdfs/general/data_pointer.rb]: Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.

File: app/pdf_generators/feedback_pdfs/general/data_pointer.rb [hunk 3]
```diff
@@ -25,11 +25,15 @@ def feedback_entries
...
+      1 => 300,
+      2 => 337
+    })
```
Comment (matthewford) [inline app/pdf_generators/feedback_pdfs/general/data_pointer.rb]: Indent the right brace the same as the first position after the preceding left parenthesis.

File: app/pdf_generators/feedback_pdfs/general/data_pointer.rb [hunk 4]
```diff
@@ -25,11 +25,15 @@ def feedback_entries
...
     render_table(table_items, {
-      0 => 100,
-      1 => 100,
-      2 => 567
+      0 => 130,
```
Comment (matthewford) [inline app/pdf_generators/feedback_pdfs/general/data_pointer.rb]: Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.

**Comment by RuslanHamidullin** [comment]
Looks perfect :+1: Waiting for travis :cactus:

**Pull request merged**